lib/internal/options.js

```diff
@@ -1,12 +1,32 @@
 'use strict';
 
 const { getOptions, shouldNotRegisterESMLoader } = internalBinding('options');
-const { options, aliases } = getOptions();
 
 let warnOnAllowUnauthorized = true;
 
+let optionsMap;
+let aliasesMap;
+
+// getOptions() would serialize the option values from C++ land.
+// It would error if the values are queried before bootstrap is
+// complete so that we don't accidentally include runtime-dependent
+// states into a runtime-independent snapshot.
+function getOptionsFromBinding() {
+  if (!optionsMap) {
+    ({ options: optionsMap } = getOptions());
```

Comment:
```suggestion
    { options: optionsMap } = getOptions();
```

Reply:
I don't think that's valid syntax? Destructuring assignments without declarations need the parentheses.

Reply:
Hmm.  But what gives here?
```console
> function test() {
...     return {hello: 1};
... }
undefined

> { hello: world } = test()
{ hello: 1 }

> { hello: world } = test();
{ hello: world } = test();
                 ^

Uncaught SyntaxError: Unexpected token '='
>
```
With the semi it's invalid, but without it it's okay?! 🤔

Reply:
Maybe it has something to do with ASI, I guess you'd need to read the spec to find out, but with the semicolon the left hand side is parsed as a block, not a destructuring assginment.